scripts/mutect2_wdl/mutect2-replicate-validation.wdl

```diff
@@ -76,10 +76,8 @@ workflow Mutect2ReplicateValidation {
 	File ref_dict
 	File? pon
 	File? pon_index
-	File? dbsnp
-	File? dbsnp_index
-	File? cosmic
-	File? cosmic_index
+	File? gnomad
```

Comment:
> Why can't we just include dbSNP, COSMIC, and gnomAD?

dbSNP is inferior to gnomAD and I don't want to give it our imprimatur, especially at the cost of enlarging our code and wdl.  If someone really wants it they can use `SelectVariants`.  In my opinion COSMIC is like a functional annotation and doesn't belong in Mutect.